src/packages.builds

```diff
@@ -10,9 +10,6 @@
     <Project Include="$(MSBuildThisFileDirectory)..\pkg\*\*.builds" Condition="'$(SkipManagedPackageBuild)' != 'true'">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
```

Comment:
Can you please follow the same model as we do in servicing https://github.com/dotnet/corefx/blob/release/1.1.0/src/packages.builds.

In that we put the full set under a BuildAllPackages condition and then explicitly list the ones we want to build.